charts/prom-label-proxy/templates/service.yaml

```
@@ -7,16 +8,19 @@ metadata:
     {{- include "prom-label-proxy.labels" . | nindent 4 }}
   {{- with .Values.service.annotations }}
   annotations:
-    {{- toYaml . | nindent 4 }}
+    {{- tpl (toYaml .) $root | nindent 4 }}
```

Is there any downside of using

```suggestion
    {{- tpl (toYaml .) $ | nindent 4 }}
```

which does not require the change in line 1?

---

Yes, should make no difference.